src/search.coffee

```diff
@@ -199,7 +177,7 @@ class Search
 
   extractWebResults: ({results} = {}) ->
     for result in results
-      id: result.id
+      id: result.id # not useful, but left for backwards compatibility
```

Comment:
Example: `https://api.cognitive.microsoft.com/api/v7/#WebPages.0` (where the appended `0` increments)